Libs/DICOM/Widgets/ctkDICOMBrowser.cpp

```diff
@@ -1319,6 +1319,8 @@ void ctkDICOMBrowser::exportSeries(QString dirPath, QStringList uids)
     destinationDir += sep;
 
     // make sure only ascii characters are in the directory path
+    // (it is safer to avoid special characters in case the files are
+    // to be more compatible with exported to external drive or network storage)
```

Comment:
`are` -> `have`
`with` -> `when`

```
    // (it is safer to avoid special characters in case the files have
    // to be more compatible when exported to external drive or network storage)
```